packages/flatpak.rb

```diff
@@ -3,7 +3,7 @@
 class Flatpak < Package
   description 'Flatpak is a system for building, distributing, and running sandboxed desktop applications on Linux.'
   homepage 'https://flatpak.org'
-  @_ver = '1.10.2'
+  @_ver = '1.15.6'
   version "#{@_ver}-1"
   license 'LGPL-2.1+'
   compatibility 'all'
```

Comment:
Additionally, please update the `source_sha256` to whatever the `sha256sum` value of the downloaded file of `https://github.com/flatpak/flatpak/releases/download/1.15.6/flatpak-1.15.6.tar.xz` is.

Reply:
Or use the git form...